### ISSUE TITLE
Resume print from Z

### ISSUE THREAD

**Issue by claytonbrown** [OutsourcedGuru/GetToDahChoppa#2]
Would make a nice plugin in octopi, i just attempted to resume a failed print at layer 900 after print edge line the extruder starts running non stop, perhaps trying to catchup from previous layers skipped

**Comment by OutsourcedGuru** [comment]
Yep. I was printing large parts in the (expensive) carbon fiber--infused filament so some of the parts were $50+. On failure, it was then important to resume the job at the affected fail point and this program did wonders for that.
